src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const mainImage = document.getElementById('largeImg');
+const secondaryImages = document.getElementById('thumbs');
+
+secondaryImages.addEventListener('click', (clickEvent) => {
+  clickEvent.preventDefault();
```

Comment:
`event` or `e` is enought name for some event type
```suggestion
secondaryImages.addEventListener('click', (event) => {
  event.preventDefault();
```